catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -145,19 +151,36 @@ public UpdateResponse version(UpdateResponse updateResponse)
         }
         setSkipFlag(updateResponse);
 
+        if (LOGGER.isTraceEnabled()) {
+            LOGGER.trace("Versioning updated metacards: {}", updateResponse.getUpdatedMetacards());
```

Comment:
Actually, it's not. But you're not smart enough to know that the doSkip check is null-safe and automatically returns if it is null.